### DISCUSSION TITLE
Suggestion for Privacy Remote Feature

### DISCUSSION THREAD

**Discussion by DanielHo4** [LizardByte/community#229] [Sunshine]
Hi all,

I am one of the new user, sometime I did not want others able to see what I am doing during remote session, however, I am not sitting next to my computer and unable to know is there anyone watching my monitor. Can Sunshine having a new feature, which able to switch off the audio and video of original display during the remote session, and resume when the active session ends?

This feature will greatly increase the privacy of the remote users.

**Comment by Ayting69** [comment]
I don't know how to do it but if you set a virtual display and disable in windows settings your physic display, they'll not see your screen i think

**Comment by DanielHo4** [comment]
Privacy mode help you to preventing leave the computer remote without lock, which is more secure. And this prevent some unintentional person from watching what user is doing while user don't even know. We cannot ensure the computer is safe when we are not close to it, so privacy mode certainly helps.

**Comment by ReenigneArcher** [comment]
I don't believe this is possible since Sunshine captures what is displayed on the screen. I would suggest using a headless solution so there is no physical monitor at all if that is a concern to you.
